nautobot/extras/utils.py

```diff
@@ -435,7 +435,7 @@ def migrate_role_data(
         queryset=queryset,
         role_model=role_model,
         role_choiceset=role_choiceset,
-        legacy_role=legacy_role_name,
+        legacy_role=legacy_role,
```

Comment:
is `legacy_role_name` now unused? The logic above checks for a special case of `not (is_choice_field or is_m2m_field)` - is that a case we need to handle specially in some other way?